plugins/storage/object/huawei-obs/src/main/java/org/apache/cloudstack/storage/datastore/lifecycle/HuaweiObsObjectStoreLifeCycleImpl.java

```diff
@@ -0,0 +1,111 @@
+package org.apache.cloudstack.storage.datastore.lifecycle;
+
+import com.cloud.agent.api.StoragePoolInfo;
+import com.cloud.hypervisor.Hypervisor.HypervisorType;
+import com.cloud.utils.exception.CloudRuntimeException;
+import com.obs.services.ObsClient;
+import com.obs.services.model.ListBucketsRequest;
+import org.apache.cloudstack.engine.subsystem.api.storage.ClusterScope;
+import org.apache.cloudstack.engine.subsystem.api.storage.DataStore;
+import org.apache.cloudstack.engine.subsystem.api.storage.HostScope;
+import org.apache.cloudstack.engine.subsystem.api.storage.ZoneScope;
+import org.apache.cloudstack.storage.datastore.db.ObjectStoreVO;
+import org.apache.cloudstack.storage.object.datastore.ObjectStoreHelper;
+import org.apache.cloudstack.storage.object.datastore.ObjectStoreProviderManager;
+import org.apache.cloudstack.storage.object.store.lifecycle.ObjectStoreLifeCycle;
+import org.apache.log4j.Logger;
+
+import javax.inject.Inject;
+import java.util.HashMap;
+import java.util.Map;
+
+public class HuaweiObsObjectStoreLifeCycleImpl implements ObjectStoreLifeCycle {
+
+    private static final Logger LOG = Logger.getLogger(HuaweiObsObjectStoreLifeCycleImpl.class);
```

Comment:
```suggestion
    protected Logger logger = LogManager.getLogger(getClass());
```
Also need to update the log4j import